core/dovecot/Dockerfile

```diff
@@ -1,14 +1,4 @@
-ARG DISTRO=alpine:3.14.2
-FROM $DISTRO as builder
-WORKDIR /tmp
-RUN apk add git build-base automake autoconf libtool dovecot-dev xapian-core-dev icu-dev
-RUN git clone https://github.com/grosjo/fts-xapian.git \
-  && cd fts-xapian \
-  && git checkout 1.2.7 \
-  && autoreconf -vi \
-  && PANDOC=false ./configure --with-dovecot=/usr/lib/dovecot \
-  && make \
-  && make install
+ARG DISTRO=alpine:3.14
```

Comment:
Why did you change this?
We use pinned versions. Please change this back to 3.14.2.

Reply:
I think it's more interesting to follow the major version (3.14) than a minor version, so the project follows the evolutions of the underlying distribution (bug fixes, no major evolution). The latest 3.14 version of alpine is 3.14.3.
I rollback and propose to open a discussion on the subject ?